src/Bicep.Cli/InvocationContext.cs

```diff
@@ -17,14 +16,14 @@ public class InvocationContext
             IAzResourceTypeLoader azResourceTypeLoader,
             TextWriter outputWriter,
             TextWriter errorWriter,
-            IFeatureProvider? features = null,
+            IFeatureProviderFactory? featureProviderFactory = null,
```

Comment:
Does this need to be nullable?

Reply:
Yes; this parameter is a backdoor for the integration tests to toggle various feature flags and set the assembly version to a static string. It is never set on contexts created in the Bicep.Cli package, which instead relies on the DI container to instantiate the factory.